irohad/torii/command_service.hpp

```diff
@@ -17,72 +17,138 @@ limitations under the License.
 #ifndef TORII_COMMAND_SERVICE_HPP
 #define TORII_COMMAND_SERVICE_HPP
 
-#include <endpoint.grpc.pb.h>
-#include <endpoint.pb.h>
 #include <iostream>
 #include <string>
 #include <unordered_map>
 #include "ametsuchi/storage.hpp"
 #include "cache/cache.hpp"
+#include "endpoint.grpc.pb.h"
+#include "endpoint.pb.h"
 #include "model/converters/pb_transaction_factory.hpp"
 #include "model/transaction_response.hpp"
 #include "torii/processor/transaction_processor.hpp"
 
 namespace torii {
-
   /**
    * Actual implementation of sync CommandService.
    */
   class CommandService : public iroha::protocol::CommandService::Service {
    public:
+    /**
+     * Creates a new instance of CommandService
+     * @param pb_factory - model->protobuf and vice versa converter
+     * @param tx_processor - processor of received transactions
+     * @param storage - storage to request transactions outside the cache
+     * @param start_tx_processing_duration - maximum time between getting a
```

Comment:
I guess this time could be also deduced from the proposal emitting time.

Reply:
One of StatusStream use cases is to do StatusStream call before Torii call to collect as much statuses as possible. Proposal emitting time will be not enough in case of more or less significant network delay.